change/@ni-nimble-components-997c813b-24bc-4cff-93cf-ebd4a16affd9.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do you plan to tackle #1090 soon? I personally think these icons really need some tooltip/ARIA text to explain their purpose.

Reply:
I wasn't planning on it, but I agree that the lack of tooltips for the icons is fairly annoying.

Reply:
I added that tech debt to Table Milestone 4 for now so that we revisit this sooner rather than later.